okio/api/okio.api

```diff
@@ -57,11 +56,6 @@ public class okio/AsyncTimeout : okio/Timeout {
 	public final fun withTimeout (Lkotlin/jvm/functions/Function0;)Ljava/lang/Object;
 }
 
-public final class okio/AsyncTimeout$Companion {
```

Comment:
Ugh, I think these snuck in by accident. I don’t believe they belong in the public API, and I don’t think anyone ever used them.